diff --git a/node_modules/babel-plugin-lodash/lib/index.js b/node_modules/babel-plugin-lodash/lib/index.js
index 16ed6be..43bde76 100644
--- a/node_modules/babel-plugin-lodash/lib/index.js
+++ b/node_modules/babel-plugin-lodash/lib/index.js
@@ -99,7 +99,7 @@ function lodash(_ref) {
 
         var node = _ref3;
 
-        if ((0, _types.isModuleDeclaration)(node)) {
+        if ((0, _types.isImportOrExportDeclaration)(node)) {
           isModule = true;
           break;
         }
